test/unifex/integration_test.exs

```diff
@@ -33,12 +37,16 @@ defmodule Unifex.IntegrationTest do
     end
   end
 
-  defp do_test_project(project, interface, language) do
+  defp run_projects_tests(project, language) do
     assert {_output, 0} =
              System.cmd("bash", ["-c", "mix test 1>&2"],
                cd: "test_projects/#{project}",
                env: [{"UNIFEX_TEST_LANG", "#{language}"}]
              )
+  end
+
+  defp do_test_project(project, interface, language) do
+    run_projects_tests(project, language)
```

Comment:
If you move generating docs to already existing testing modules, this change will become unnecessary

Reply:
Ok, applied